demisto_sdk/tests/test_files/conf.json

```diff
@@ -46,6 +46,9 @@
     "nightly_packs": [
         "CommonScripts"
     ],
+    "selected_nightly_tpb": [
+        "test test"
```

Comment:
what is this used for?

Reply:
Now this field is required, so i need to add it to old tests (in the test data)  in order to pass